src/services/compare/handlers/sendDoesNotExistAlert.js

```diff
@@ -1,11 +1,18 @@
 import {
-  sendAlert,
+  sendTemplatedEmail,
   doesSecretExist,
   getSecretsValue,
   putLogsEvent,
   trackError,
 } from "../../../libs";
 
+const Templates = {
+  SendNoMatchTemplate: "getRecordDoesNotMatchParams",
+  SendNoMatchTemplateAB: "getRecordDoesNotExistParamsAB",
+  SendNoMatchTemplateChp: "getRecordDoesNotExistParamsChp",
+  SendNoMatchTemplateChpAB: "getRecordDoesNotExistParamsChpAB",
```

Comment:
i think the AB bit is kind of confusing. I wonder if we can be more specific, like 'SendNoMatchTemplateInitial` and 'SendNoMatchTemplateFollowUp` im not sold on the names but maybe something that is easier to understand from the template name